solidity/dashboard/src/components/coverage-pools/LearnMoreBanner.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import Banner from "../Banner"
+import * as Icons from "../Icons"
+import NavLink from "../NavLink"
+
+const LearnMoreBanner = ({ onClose }) => {
+  return (
+    <Banner className="banner banner--coverage-pools">
+      <Banner.CloseIcon onClick={onClose} />
+      <div className="flex row space-between full-center">
+        <Banner.Icon icon={Icons.CoveragePool} />
+        <Banner.Title className="h3 text-white">
+          Deposit KEEP in the coverage pool to
+          <br />
+          secure the network and earn rewards.
+        </Banner.Title>
+        <NavLink
+          to="coverage-pools/how-it-works"
```

Comment:
I think you missed `/` at the beginning here. But I think that we will actually put it in a variable when we create `How it works` page so this comment might be pointless.